src/main/java/org/sonarqube/gradle/SonarTask.java

```diff
@@ -128,7 +131,12 @@ public void run() {
     ScannerEngineBootstrapper scanner = ScannerEngineBootstrapper
       .create("ScannerGradle", getPluginVersion() + "/" + GradleVersion.current())
       .addBootstrapProperties(mapProperties);
-    try (ScannerEngineFacade engineFacade = scanner.bootstrap()) {
+    try (ScannerEngineBootstrapResult result = scanner.bootstrap()) {
+      // implement behavior according to SCANJLIB-169
+      if(!result.isSuccessful()){
+        throw new AnalysisException("The scanner boostrapping has failed! See the logs for more details.");
+      }
+      ScannerEngineFacade engineFacade = result.getEngineFacade();
       boolean analysisIsSuccessful = engineFacade.analyze(new HashMap<>());
```

Comment:
Renaming the variable might be useful for clarity here
```suggestion
    try (ScannerEngineBootstrapResult bootstrapping = scanner.bootstrap()) {
      // implement behavior according to SCANJLIB-169
      if(!bootstrapping.isSuccessful()){
```

Reply:
:+1: